### DISCUSSION TITLE
Fork project make it private

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#124131] [Repositories]
### Select Topic Area

Product Feedback

### Body

Is there a way for a project that has been forked to make it private?

He won't let me do it.
<img width="845" alt="Screenshot 2024-05-16 alle 16 13 55" src="https://github.com/community/community/assets/20476002/9cd2881d-3188-44d0-98bc-2cd9fb5835df">

**Comment by davevad93** [comment]
Hi @Angelk90 ,

It is not possible to make a forked repository private by changing the visibility settings, however there's a kind of workaround, check this out:

- [GitHub: How to make a fork of public repository private?](http://stackoverflow.com/questions/10065526/github-how-to-make-a-fork-of-public-repository-private)
